docs/getting_started/dbt-airflow-concepts.rst

```diff
@@ -0,0 +1,34 @@
+.. _dbt-airflow-concepts:
+
+Similar dbt & Airflow concepts
+==============================
+
+While dbt is an open source tool for data transformations and analysis, using SQL, Airflow focuses on being a platform
+for the development, scheduling and monitoring of batch-oriented workflows, using Python. Although both tools have many
+differences, they also share similar concepts.
+
+This page aims to list some of these concepts and help those
+who may be new to Airflow or dbt and are considering to use Cosmos.
+
+
++----------------+--------------+---------------------------------------------------------------------------------+-----------------------------------------------------------------------------+--------------------------------------------------------------------------------------+
+| Airflow naming | dbt naming   | Description                                                                     | Differences                                                                 | References                                                                           |
++================+==============+=================================================================================+=============================================================================+======================================================================================+
+| DAG            | Workflow     | Pipeline (Direct Acyclic Graph) that contains a group of steps                  | Airflow expects upstream tasks to have passed to run downstream tasks.      | https://airflow.apache.org/docs/apache-airflow/2.7.1/core-concepts/dags.html         |
+|                |              |                                                                                 | dbt can run a subset of tasks assuming upstream tasks were run.             | https://docs.getdbt.com/docs/introduction                                            |
```

Comment:
I would maybe put this difference in the task section?

Reply:
The difference talks about airflow task dependencies, and this is discussing DAGs

Reply:
@harels I see tasks as individual nodes (vertices), whereas DAGs represent a group of nodes and edges - and how the nodes relate to each other - so I believe this comment is more suitable in the DAGs/workflows. WDYT? If you feel strongly about it, we can still change before the release 1.1.1.